tests/sdk/conftest.py

```
@@ -0,0 +1,72 @@
+    http_client = httpx.Client(
+        transport=client._transport,  # pyright: ignore
+        base_url=str(client.base_url),
+        headers=client.headers,
+    )
+
+    honcho_client = AsyncHoncho(
+        workspace_id="sdk-test-workspace-async",
+        async_http_client=async_http_client,
+        http_client=http_client,
+    )
+    return honcho_client
```

Ruff (0.11.9)

Unnecessary assignment to `honcho_client` before `return` statement

Remove unnecessary assignment

(RET504)